packages/ui/src/memberships/hooks/useMyRoleIds.ts

```diff
@@ -0,0 +1,16 @@
+import { useMemo } from 'react'
+
+import { useMyMemberships } from '@/memberships/hooks/useMyMemberships'
+import { useGetWorkerIdsQuery } from '@/working-groups/queries'
```

Comment:
It should go to `@/working-groups` since it's a working group related. A broader search is `useMyWorkers()`.

I wonder if there's a need for this hook, or couldn't we re-use `useMyWorkers()` – I think that this is not the case here probably. If those hooks are used on different pages it makes sense to have two of them.

Reply:
I just wanted to minimize the data payload - not sure if it's all that needed, but Worker fields seem to be the among the heftiest query results at the moment